tests/Frameworks/WordPress/Version_4_8/wp-config.php

```diff
@@ -80,6 +80,9 @@
  */
 define('WP_DEBUG', false);
 
+/** Disabilita la funzione interna Wp-Cron **/
+define('DISABLE_WP_CRON', true);

```

Comment:
I had to disable cron jobs because I had a different behavior locally and in CI. Created a [ticket](https://datadoghq.atlassian.net/browse/APMPHP-25?atlOrigin=eyJpIjoiYWYyYjE3NmQ2ZTE1NGM4MWJlMTNiZDExM2NiNzdkNjQiLCJwIjoiaiJ9) to keep track of this and investigate with lower priority